Sweeper/Models/PinComment.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PinComment.swift
+//  Sweeper
+//
+//  Created by Wuming Xie on 10/20/17.
+//  Copyright © 2017 team11. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+class PinComment: PFObject {
+    
+    @NSManaged var user: User?
+    @NSManaged var comment: String?
+    
+}
+
+extension PinComment: PFSubclassing {
+    static func parseClassName() -> String {
+        return "PinComment"
+    }
+}
```

Comment:
Did you create this class on the Parse dashboard? It might create automatically, but I don't see any comment records.

Reply:
No I haven't yet. I still need to actually connect this to the backend.